src/test/Makefile-client.am

```diff
@@ -234,7 +235,7 @@ ceph_test_rados_api_aio_CXXFLAGS = $(UNITTEST_CXXFLAGS)
 bin_DEBUGPROGRAMS += ceph_test_rados_api_aio
 
 ceph_test_rados_api_list_SOURCES = test/librados/list.cc
-ceph_test_rados_api_list_LDADD = $(LIBRADOS) $(UNITTEST_LDADD) $(RADOS_TEST_LDADD)
+ceph_test_rados_api_list_LDADD = $(LIBRADOS) $(UNITTEST_LDADD) $(RADOS_TEST_LDADD) $(CEPH_GLOBAL)
```

Comment:
@tchaikov 
Below the error that I get if I remove CEPH_GLOBAL from the LDADD list
test/librados/ceph_test_rados_api_list-list.o: In function `main':
/usr/srcs/Ceph/work/ceph/src/test/librados/list.cc:973: undefined reference to `global_init(std::__1::vector<char const*, std::__1::allocator<char const*> >*, std::__1::vector<char const*, std::__1::allocator<char const*> >&, unsigned int, code_environment_t, int, char const*, bool)'
/usr/srcs/Ceph/work/ceph/src/test/librados/list.cc:974: undefined reference to `g_ceph_context'